docs/BR.md

```diff
@@ -1099,8 +1103,40 @@ No stipulation.
 
 ### 4.3.1 CA actions during certificate issuance
 
+#### 4.3.1.1 Manual authorization of certificate issuance for Root CAs
+
 Certificate issuance by the Root CA SHALL require an individual authorized by the CA (i.e. the CA system operator, system officer, or PKI administrator) to deliberately issue a direct command in order for the Root CA to perform a certificate signing operation.
 
+#### 4.3.1.2 Linting of to-be-signed Certificate content
+
+Due to the complexity in implementing the certificate-formatting standards of these Requirements when issuing publicly-trusted Certificates, before a Certificate or Precertificate [RFC 6962] is issued, effective 2024-09-15 the CA SHOULD implement a Linting process to test the technical conformity of the to-be-signed Certificate with these Requirements. Effective 2025-03-15, the CA SHALL implement a Linting process to test the technical conformity of the to-be-signed Certificate with these Requirements.
```

Comment:
I think the current language is unclear whether linting of the TBS final Certificate is required if TBS linting of the corresponding Precertificate is performed.

I don't think that TBS final Certificate linting is needed as long as the CA has a technical control to verify that the TBS content of the Precertificate matches the content of the TBS final certificate. Absent that, it would be good to lint the TBS final Certificate.

Thoughts?

Reply:
I'm not sure about that one. Perhaps @robstradling can assist?

Reply:
I agree with Corey's statement - "I don't think that TBS final Certificate linting is needed as long as the CA has a technical control to verify that the TBS content of the Precertificate matches the content of the TBS final certificate" - although I would change "matches" to "corresponds to".  (The TBS objects are not identical; one has the poison extension, whilst the other has the SCT List extension).

I've had a go at wordsmithing this paragraph...

```suggestion
Due to the complexity involved in implementing the Certificate Profiles defined by these Requirements, it is considered best practice for the CA to implement a Linting process to test the technical conformity of each to-be-signed artifact prior to signing it.  When a Precertificate has undergone Linting, it is not necessary for the corresponding to-be-signed Certificate to also undergo Linting, provided that the CA has a technical control to verify that the to-be-signed Certificate corresponds to the to-be-signed Precertificate in the manner described by RFC 6962, Section 3.2.
Effective 2024-09-15, the CA SHOULD implement such a Linting process.
Effective 2025-03-15, the CA SHALL implement such a Linting process.
```

WDYT?

Reply:
I think this language is good. Two comments:

1. "matches" is used in the same context in section 4.9.10, so there is consistency in using that term. That being said, "corresponds to" is fine as well. Maybe change 4.9.10 if we go with "corresponds to"?
2. "Certificate Profiles" are not defined by these Requirements, but rather they are documents/configurations that yield Certificates that comply with the requirements in section 7. Given this, I suggest changing the first clause of the first sentence to:
```
Due to the complexity involved in implementing Certificate Profiles that conform to these Requirements, ...
```

Reply:
Thanks @CBonnell.  I support changing 4.9.10 to get rid of "matches".

FYI, there's at least one more example of existing language that's consistent with this (i.e., "corresponds to" without "matches") - see Section 7.1.2.9, which says _"The existence of a signed Precertificate can be treated as evidence of a corresponding Certificate also existing,"_

With the current wording of the Definition of "Certificate Profiles", I have to +1 to your suggested change.  However, given the Definition of "Technically Constrained Subordinate CA Certificate" (see below, emphasis mine), I wonder if we need to tweak the Definition of "Certificate Profiles" instead...

_"Technically Constrained Subordinate CA Certificate: A Subordinate CA certificate which uses a
combination of Extended Key Usage and/or Name Constraint extensions, as **defined within the
relevant Certificate Profiles of this document**, to limit the scope within which the Subordinate CA
Certificate may issue Subscriber or additional Subordinate CA Certificates."_

WDYT?

Reply:
These are good call-outs. I think we want to retain the current definition of Certificate Profiles, as it was created for a specific purpose tied to a normative requirement (logging requirements in 5.4.1). Any misuse of the defined term should be corrected instead of shoehorning the definition to make it fit in all its current usages.

Reply:
I believe fe2a2d333b6d761d1a62fda4f4317425f9ec8e3a improves the language for this particular ballot. I suggest @robstradling @CBonnell to create an issue regarding the "Certificate Profiles" definition and possible other improvements to handle in a separate ballot. Please let me know if you are ok with this to resolve this conversation.

Reply:
@dzacharo That works for me.

Reply:
https://github.com/cabforum/servercert/issues/526